shacl12-overview/index.html

```diff
@@ -346,30 +346,118 @@
+			<h2>What's New in SHACL 1.2</h2>
 			<p>
-				These shapes are available in the RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
+				<em>coming soon</em>
 			</p>
 		</section>
 
-		<section id="governance">
-			<h2>Governance</h2>
-			<p class="todo">TODO: This section will describe the governance arrangements in place for the continued maintenance of this NOTE.</p>
+		<section id="specifications">
+			<h2>Specifications</h2>
+			<p>
+				SHACL 1.2:
+			</p>
+			<ol>
+				<li>SHACL 1.2 Overview (this document)
+					<ul><li>overviews the set of SHACL specifications</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-core/">SHACL 1.2 Core</a>
+					<ul><li>defines the Core of SHACL</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-sparql/">SHACL 1.2 SPARQL Extensions</a>
+					<ul><li>defines SPARQL-related extensions of the SHACL</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-node-expr/">SHACL 1.2 Node Expressions</a>
+					<ul><li>defines graph expressions used to determine focus nodes SHACL</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-inf-rules/">SHACL 1.2 Inference Rules</a>
+					<ul><li>defines SHACL's methods of rule-based inference</li></ul>
+				</li>
+				<li>
+					<s><a href="https://w3c.github.io/data-shapes/shacl12-ui/">SHACL 1.2 UI</a></s>
+					<ul><li>defines SHACL's use for User Interface generation</li></ul>
+				</li>
+				<li>
+					<s><a href="https://w3c.github.io/data-shapes/shacl12-profiling/">SHACL 1.2 Profiling</a></s>
+					<ul><li>defines the use of SHACL for profiling data, including SHACL data</li></ul>
+				</li>
+			</ol>
+		</section>
+
+		<section id="x">
+			<h2>Section X</h2>
+			<p class="todo">TODO: This section will describe...</p>
+		</section>
+
+		<section id="shacl-shacl" class="appendix informative">
+			<h2>SHACL-SHACL</h2>
+			<p>
+				The <a href="https://www.w3.org/groups/wg/data-shapes/">Data Shapes Working Group</a> maintains SHACL graphs that can be used to validate other SHACL Shapes graphs, called "SHACL-SHACL". There is more than one to allow for the validation of subsets of the total body of SHACL elements defined in the various SHACL specifications.
+			</p>
+			<p>
+				Each SHACL-SHACL graph corresponds to the scope of individual SHACL specifications and are available as RDF resources:
+			</p>
+			<ul>
+				<li>
+					<a href="">SHACL-SHACL all</a> - <em>the union of all other SHACL-SHACL graphs</em>
+				</li>
+				<li>
+					<a href="">SHACL-SHACL Core</a>
+				</li>
+				<li>
+					<a href="">SHACL-SHACL SPARQL</a>
+				</li>
+				<li>
+					<a href="">SHACL-SHACL Node Expressions</a>
+				</li>
+				<li>
+					<a href="">SHACL-SHACL Inference Rules</a>
+				</li>
+				<li>
+					<a href="">SHACL-SHACL UI</a>
+				</li>
+				<li>
+					<a href="">SHACL-SHACL Profiling</a>
```

Comment:
A SHACL-SHACL graph for profiling?  I won't object at this point, as I know there is a variety of interpretations of "Profile" among the working group.  I'll just note that my own interpretation of "Profile" was that all of the above files in this list correspond 1-to-1 with profiles.  I'm curious if you think this file could do any more than would be necessary to specify in SHACL-SHACL Core?

Explanation not needed on GitHub, I'm happy to discuss in a working-group call.

Reply:
Yes, this will need to be beaten out in specific SHACL Profiling subgroup meetings, so let's just leave this in here for now and revise as the subgroup fixes scope.